src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp

```diff
@@ -594,8 +594,8 @@ class boss_freya : public CreatureScript
                 const uint32 summonSpell[2][4] =
                 {
                               /* 0Elder, 1Elder, 2Elder, 3Elder */
-                    /* 10N */    {62950, 62953, 62955, 62957},
-                    /* 25N */    {62952, 62954, 62956, 62958}
+                    /* 10N */    {62957, 62955, 62953, 62950},
+                    /* 25N */    {62958, 62956, 62954, 62952}
```

Comment:
did you checked sniffs? i guess it's the same problem like thorim (wrong db content)